src/utils/saml.ts

```diff
@@ -0,0 +1,21 @@
+import { DOMParser } from "xmldom";
+
+export interface ISamlResponse {
+  RelayState: string;
+  SAMLResponse: string;
+}
+
+export const getSamlIssuer = (body: ISamlResponse): string | undefined => {
+  const decodedResponse =
+    body && body.SAMLResponse
```

Comment:
given the type declaration of this function, `body` is always defined, so perhaps `body` should be of type `ISamlResponse | undefined` ?

Reply:
Could be the type of `body` parameters `unknown`?

Reply:
yes, it would be much better, then you can use io-ts do decode it to a typed object

Reply:
fixed in 1145b45